android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMManager.java

```diff
@@ -2294,12 +2297,31 @@ public boolean shouldOverrideUrlLoading(WebView view, String url) {
         pageLoaded(view, url);
       } else if (url.indexOf("hideKeyboard") >= 0) {
         if (KMTextView.activeView != null && KMTextView.activeView.getClass() == KMTextView.class) {
+          InAppKeyboard.dismissHelpBubble();
           KMTextView textView = (KMTextView) KMTextView.activeView;
           textView.dismissKeyboard();
         }
       } else if (urlCommand.getQueryParameter("globeKeyAction") != null) {
+        InAppKeyboard.dismissHelpBubble();
+        InAppKeyboard.setShouldShowHelpBubble(false);
+
+        // // Original had logic that would shortcut this in some scenarios, but I'm unclear as to its real purpose.
+        // if (!InAppKeyboard.isHelpBubbleEnabled) {
+        //   return false;
+        // }
+
+        // Globe key has been used; disable the internal preference setting.
+        SharedPreferences prefs = appContext.getSharedPreferences(appContext.getString(R.string.kma_prefs_name), Context.MODE_PRIVATE);
+        SharedPreferences.Editor editor = prefs.edit();
+        editor.putBoolean(KMManager.KMKey_ShouldShowHelpBubble, false);
+        editor.commit();
```

Comment:
Can we refactor this into a function rather than burying it in the globe key action?